lib/plugins/create/templates/plugin/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// https://github.com/serverless/serverless/blob/master/docs/04-extending-serverless/01-creating-plugins.md
+class ServerlessPlugin {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      deploy: {
```

Comment:
Instead of deploy can you change it to something like `hello`. This makes sure users don't run into an issue as they have two deploy functions enabled.